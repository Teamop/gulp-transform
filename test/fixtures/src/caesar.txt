I am constant as the northern star...