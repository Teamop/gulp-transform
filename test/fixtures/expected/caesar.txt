I am constant as the northern star...
I am constant as the northern star...
I am constant as the northern star...
I am constant as the northern star...